### PR TITLE
#408 Added DyUsage class and implemented DyUser.usage() method

### PR DESCRIPTION
**Pull request by carlosmiranda**

### CODE REVIEW

**Comment by davvd** [comment]
Thanks, let me find someone who can review this pull request

**Comment by davvd** [comment]
@krzyk it's yours, please review

File: src/test/java/com/thindeck/dynamo/DyUserTest.java [hunk 1]
```diff
@@ -96,4 +98,20 @@ public void canGetRepos() throws Exception {
             Matchers.<Repo>iterableWithSize(1)
         );
     }
+
+    /**
+     * DyUser can return usage associated with the user info.
```
Comment (krzyk) [inline src/test/java/com/thindeck/dynamo/DyUserTest.java]: Maybe just: *DyUser can return usage info*

File: src/test/java/com/thindeck/dynamo/DyUsageTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+/**
+ * Test case for {@link DyUsage}.
+ *
+ * @author Carlos Miranda (miranda.cma@gmail.com)
+ * @version $Id$
+ * @since 0.5
+ */
+public final class DyUsageTest {
+    /**
+     * DyUsage can return its associated user.
```
Comment (krzyk) [inline src/test/java/com/thindeck/dynamo/DyUsageTest.java]: Maube: *DyUser can return its user*

**Comment by krzyk** [comment]
@carlosmiranda two comments from me

**Comment by carlosmiranda** [comment]
@krzyk Made commits related to your comments. Please check again.

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by carlosmiranda** [comment]
@yegor256 Can you confirm the merge?

**Comment by yegor256** [comment]
@rultor merge pls

**Comment by carlosmiranda** [comment]
@rultor try to merge again please

**Comment by davvd** [comment]
@krzyk thanks, paid, **17 mins** to your account, payment ID is `53353606`. extra minutes for review comments (c=2). +17 added to your rating, at the moment it is: [+5178](http://www.netbout.com/b/35035?open=rating)

**Comment by davvd** [comment]
@rultor deploy now pls

**Pull request merged**